tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -2404,6 +2428,8 @@ <h2>Show similarity to selected datapoint</h2>
           readOnly: true,
         },
         // Timer for debounding of refreshing dive/stats after data changes.
+        refreshPDPlotTimer_: Number,
+        // Timer for debounding of refreshing dive/stats after data changes.
```

Comment:
looks like i had a typo in my comment for the other timer. should be "debouncing"

Reply:
done.